lib/js/components/Pagination/Pagination.vue

```diff
@@ -54,7 +54,8 @@
 						:min="FIRST_PAGE_NUMBER"
 						:step="1"
 						:max="lastPage"
-						@input="onInputValueChange"
+						@blur="onInputValueConfirmed"
```

Comment:
`blur` will trigger unnecesary events, whe someone didn't change the value. Try using `@change`: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event